packages/compass-telemetry/src/telemetry-events.ts

```diff
@@ -79,7 +82,7 @@ export type ConnectionScopedProperties = {
   /**
    * The id of the connection associated to this event.
    */
-  connection_id: string;
+  connection_id?: string;
```

Comment:
looks like `connection_id` is not guaranteed for all the connection scoped events

Reply:
Any idea which ones? Sounds dodgy.

Reply:
Screen is one that is hard to fix, and then we found some others when connecting from CLI without an existing id